src/services/config/config.service.ts

```diff
@@ -37,6 +44,27 @@ export default class ConfigService {
       throw new Error(`No config for network key: ${key}`);
     return configs[key];
   }
+
+  public get rpc(): string {
+    return template(this.network.rpc, {
```

Comment:
I've not seen this es6 dynamic template before, what is it doing?

Reply:
Just a regex replace. I'll actually move this function into our utils instead of using it from npm. Or add something more vue like (my worry about using vue here was it might create a dependency loop).